05_run_ocp.sh

```diff
@@ -107,16 +107,27 @@ ssh -o "StrictHostKeyChecking=no" core@$IP sudo modprobe iscsi_tcp
 ssh -o "StrictHostKeyChecking=no" core@$IP sudo mkdir /run/httpd
 
 # Build and start the ironic container
-cat ironic/Dockerfile | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=Dockerfile
+"StrictHostKeyChecking=no" core@$IP sudo mkdir -p /home/core/ironic
```

Comment:
Missing "ssh -o " here

Reply:
whoops, thanks will fix - side-note, why do you use cat foo | ssh instead of scp here?

Reply:
Not sure, it was either because I was originally going to put the file in ~root which is easier with sudo dd OR because I was too lazy to "man scp" to see if "-o StrictHostKeyChecking=no" worked with scp...., either way scp would be fine